src/components/Dropdown/Dropdown.story.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import { generateVariantsFor } from '../../utils/propsVariants';
+import Dropdown from './Dropdown';
+import iconNames from '../Icon/Icons';
+
+const stories = storiesOf('Dropdown', module);
+
+stories.add(
+  'Dropdown',
+  () => {
+    const variants = generateVariantsFor({
+      topText: [null, '', 'Resources'],
+      topIcon: [null, Object.keys(iconNames)[0]],
+      items: [
+        [],
+        [
+          { value: 'Data Resource Protal', icon: 'file-size', onClick: () => {} },
+          { value: 'Website', icon: 'website', onClick: () => {} },
+          { value: 'Studies and Access', icon: 'study', onClick: () => {} },
+          { value: 'Support', icon: 'info', onClick: () => {} },
+          { value: 'Contact', icon: 'email', onClick: () => {} },
+        ],
+      ],
+    });
+    const items = [
+      { value: 'Data Resource Protal', icon: 'file-size', onClick: () => {} },
+      { value: 'Website', icon: 'website', onClick: () => {} },
+      { value: 'Studies and Access', icon: 'study', onClick: () => {} },
+      { value: 'Support', icon: 'info', onClick: () => {} },
+      { value: 'Contact', icon: 'email', onClick: () => {} },
+    ];
+    return (
+      <section>
+        <Dropdown
+          className="hide-in-tests"
+          topText={text('Dropdown Lable', 'Resources')}
+          topIcon={select('Dropdown Icon', Object.keys(iconNames), 'resources')}
+          items={items}
+        />
+        <div className="show-in-tests hidden">
+          {variants.map(props => (
+            <Dropdown className="m-20 percy-min-width" {...props} />
```

Comment:
move `percy-min-width` to the show-in-tests div, otherwise it will make our menus 960px wide